lib/logstash/codecs/protobuf.rb

```diff
@@ -0,0 +1,189 @@
+# encoding: utf-8
+require 'logstash/codecs/base'
+require 'logstash/util/charset'
+require 'protocol_buffers' # https://github.com/codekitchen/ruby-protocol-buffers
+
+class LogStash::Codecs::Protobuf < LogStash::Codecs::Base
```

Comment:
Hallo and thanks!
I added the documentation. I'm not sure how to preview the doc though, I didn't recognise the formatting. Could you give me a hint on how to debug the documentation to see what it would look like when rendered? Thanks a lot!